### PR TITLE
Update frontpage to include new project description

### PR DESCRIPTION
**Pull request by mrego**
This also update the site tagline and description.

### CODE REVIEW

File: _includes/hero.html [hunk 1]
```diff
@@ -1,13 +1,28 @@
-<section class="container hero is-primary is-bold is-medium">
+<section class="container is-primary is-bold is-medium">
   <div class="hero-body columns is-vcentered">
+    <div class="column">
+      <img src="{{ site.logoNegative | url }}" alt="Servo logo" style="padding: 1rem;" />
```
Comment (mukilan) [inline _includes/hero.html:4]: The large logo looks very odd to me. Maybe we can have a screenshot of Servo window here?

Comment (mukilan) [reply _includes/hero.html:4]: Did a quick mockup using one of the existing screenshots:
servo-home-alternate

Comment (mrego) [reply _includes/hero.html:4]: Can you put a screenshot of how it looks to you?

Comment (mrego) [reply _includes/hero.html:4]: This is how it looks for me.

Screenshot with servo logo

Comment (mrego) [reply _includes/hero.html:4]: Ok, I got what you meant, that there's a lot of black background there, right. I guess we can try something like you suggest.

Comment (mrego) [reply _includes/hero.html]: I have added a screenshot, though I've also kept the big logo as I kind of like it.

Screenshot of servo.org frontpage, with logo description and screenshot

How does it look like now?

Comment (mukilan) [reply _includes/hero.html]: Hmm, it still seems odd and now more 'noisy' as well, compared to [this](https://user-images.githubusercontent.com/251518/267542472-16d206cf-7678-473e-9be8-b0592374ad51.png). The one I linked to feels more clean and polished. However, that might be just me, so maybe we can decide based on input from others as well?

Comment (sagudev) [reply _includes/hero.html]: IMO smaller logo would be better. Also old screenshot looks better at top edge.

Comment (mrego) [reply _includes/hero.html]: Ok, I'll remove the big logo :innocent:. Something like:
Screenshot of servo.org frontpage without big logo

Old screenshot is using legacy layout, I'm not sure we want to show that one. Another option would be to not show a screenshot until it looks better and put there the YouTube video. And have a few more blog posts. For example:
Screenshot of servo.org frontpage without screenshot and with YouTube video and 4 blog posts

WDYT?

Comment (sagudev) [reply _includes/hero.html]: Switching to old wiki layout (vector without 2022) looks good also on new layout engine:
slika

I just do not know how to make screenshot with fake title bar like https://marketplace.visualstudio.com/items?itemName=adpyke.codesnap

Comment (sagudev) [reply _includes/hero.html]: Somebody that has access to mac can make screenshot for https://en.wikipedia.org/wiki/Servo_(software)?useskin=vector

Comment (atbrakhi) [reply _includes/hero.html]: I like this version better:
image

Comment (mrego) [reply _includes/hero.html]: Thanks for the feedback, and yeah we can get a better screenshot. I guess we can discuss adding a screenshot in a next PR and so far go with the option that only adds the new project description.

**Pull request merged**